setup.py

```diff
@@ -69,6 +69,9 @@
             =======================
 
             See http://jacquev6.github.com/PyGithub"""),
+        install_requires=[
+          "requests==2.9.1"
```

Comment:
Should you really ==2.91? Presumably this code isn't doing anything special that would work in older versions of requests.